helm-charts/konk-service/templates/configmap.yaml

```
@@ -50,9 +50,14 @@ data:
     metadata:
       name: {{ .Values.version }}.{{ .Values.group }}
     spec:
+      {{- if eq .Values.insecureSkipTLSVerify false }}
       caBundle: {{` {{ CERT }} `}}
+      {{- end }}
       group: {{ .Values.group }}
       groupPriorityMinimum: 1000
+      {{- if eq .Values.insecureSkipTLSVerify true }}
+      insecureSkipTLSVerify: {{ .Values.insecureSkipTLSVerify }}
+      {{- end }}
```

Now that you're not trying to match null values as true, this logic could be simplified:
```suggestion
      {{- if .Values.insecureSkipTLSVerify }}
      insecureSkipTLSVerify: {{ .Values.insecureSkipTLSVerify }}
      {{- else }}
      caBundle: {{` {{ CERT }} `}}
      {{- end }}
      group: {{ .Values.group }}
      groupPriorityMinimum: 1000
```

---

Done